apps/perf-test/tasks/flamegraph/helpers.js

```diff
@@ -0,0 +1,17 @@
+// TODO: test regex on mac, linux and windows
+const frameworkName = new RegExp(
+  /~([a-zA-Z]{1}|[a-zA-Z]{2}|[a-zA-Z]{1}|[a-zA-Z]{1}\.[a-zA-Z]{1})[\. ]|~render |~\(anonymous\)|\(unknown\)|\~webpack/
+);
+const systemName = new RegExp(/\(C\+\+\)|\(lib\)/);
```

Comment:
Probably don't need the constructor here either
```suggestion
const systemName = /\(C\+\+\)|\(lib\)/;
```